README.md

```diff
@@ -51,3 +51,28 @@ scrape_configs:
         replacement: 127.0.0.1:9275 # Nrpe exporter.
 
 ```
+
+## SSL support
+
+Add query parameter `ssl=1` to use NRPE over SSL.
```

Comment:
Let's include a partial bit of config for this. Query is also ambiguous here, I'd say URL.